.github/workflows/publish.yaml

```diff
@@ -2,6 +2,7 @@ name: publish
 on:
   push:
     tags:
+    - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
should the rest of the patterns go away so the `v` isn't included by mistake?

Reply:
yeah, probably.  updated